### PR TITLE
Feature/unified message

### PR DESCRIPTION
**Pull request by tombrake**
The purpose of this is to get the same information across both mercury and sns without breaking any existing lambdas. 

To do so the fields sent are now synced in the 'record' variable, which is an additive and non breaking change for any SNS subscribers. The EventMessage for mercury now uses the record as its event, with all of the original fields and a few more.

### CODE REVIEW

**Comment by niahmiah** [comment]
We need to avoid changing the database record, but add it to the message only. Also, we can't break existing clients using the mercury implementation. Please take another shot at those things :)

**Comment by tombrake** [comment]
> We need to avoid changing the database record, 

Why include a schema version in the db record then? 

>Also, we can't break existing clients using the mercury implementation

Explain why this is breaking? There were no removed fields only additive. Anyone referencing js properties by ordinals deserves what they get.

**Comment by tombrake** [comment]
Any commentary on the branding change though? Seems strange to do it two ways

File: tasks/retrieve-content.js [hunk 1]
```diff
@@ -38,82 +38,90 @@ console.log(`- Query Content Types: ${JSON.stringify(config.get('queryContentTyp
...
+                            const recordMessage = JSON.stringify(record),
+                                snsMessage = JSON.stringify(Object.assign({}, record,
+                                    {
+                                        source: doc.dataSource,
+                                        section: doc.section,
+                                        id: doc.id,
+                                    }
+                            ));
+
+                            amqp.channel.publish(exchangeName, key, new Buffer(recordMessage));
```
Comment (tombrake) [inline tasks/retrieve-content.js]: There was a question about this. Does it need to match the db record or would using the snsMessage here be okay?

**Comment by tombrake** [comment]
Resubmitted without change to record schema or eventMessage.

**Comment by tombrake** [comment]
actualizado 

Branding uses attributes first, doc.branding as fallback.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cnn-town-crier",
-  "version": "2.5.0",
+  "version": "2.5.1",
```
Comment (jamsyoung) [inline package.json]: don't change the package.json version number in branches.  The version is incremented when a release is created which may contain more than a single code change (PR).

**Pull request closed without merging**